docs/agents/prompts-redaccion.md

```
@@ -0,0 +1,284 @@
+**En Claude:**
+1. Adjuntar los archivos relevantes (p. ej. `configuracion.tex`, el capítulo
+   existente si se quiere mejorar)
+2. Pegar el prompt
+
+---
+
+## Prompts de estructura
+
+### P01 — Proponer estructura de capítulo
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)